app/models/friendship.rb

```diff
@@ -0,0 +1,14 @@
+# A class for handling contacts, when one contact is created the other way is too
```

Comment:
I think `when one contact is created the other way is too` isn't good English, not quite sure if my suggestion is better :sweat_smile: 
```suggestion
# A class for handling contacts which ensures that the association is always symmetrical
```